components/logs.go

```diff
@@ -0,0 +1,65 @@
+package components
+
+import (
+	"encoding/json"
+	"io"
+	"net/url"
+	"strings"
+
+	"github.com/flanksource/commons/http"
+
+	"github.com/flanksource/incident-commander/api"
+	"github.com/flanksource/incident-commander/db"
+)
+
+func GetLogsByComponent(componentID, logStart string) (api.LogsResponse, error) {
```

Comment:
This should be start and end, even though we have implemented the end functionality yet